src/subcommand/server.rs

```diff
@@ -1583,11 +1583,12 @@ impl Server {
         };
       };
 
-      if let Some(delegate) = inscription.delegate() {
-        inscription = index
-          .get_inscription_by_id(delegate)?
-          .ok_or_not_found(|| format!("delegate {inscription_id}"))?
-      }
+      let inscription_override = inscription
+        .delegates()
+        .iter()
+        .find_map(|delegate| index.get_inscription_by_id(*delegate).unwrap_or(None));
```

Comment:
We should handle the error here instead of using `unwrap_or`. All errors must be handled >:]